backend/Cargo.toml

```diff
@@ -7,7 +7,7 @@ rust-version = "1.85"
 default-run = "abacus"
 
 [features]
-default = ["openapi", "dev-database"]
+default = ["openapi", "dev-database", "embed-typst"]
```

Comment:
`embed-typst` as a default feature defeats the purpose of this feature?

Reply:
This is still a WIP :-) I'll remove this.